tests/test_manifest.py

```diff
@@ -6,8 +7,15 @@
 from smartsim.control import Manifest
 from smartsim.database import Orchestrator
 from smartsim.error import SmartSimError
+from smartsim.exp.ray import RayCluster
 from smartsim.settings import RunSettings
 
+try:
+    import ray
+except ImportError:
```

Comment:
``sys.modules`` doesn't work sometimes for me.

see how I did it here -> https://github.com/Spartee/SmartSim/blob/c00d8b301ea9dfb99dda4faba1dd83c3212ee739/tests/backends/test_onnx.py#L9